jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshHandler.java

```diff
@@ -0,0 +1,222 @@
+package org.jbehavesupport.core.ssh;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections4.keyvalue.MultiKey;
+import org.apache.commons.collections4.map.LRUMap;
+import org.apache.commons.collections4.map.MultiKeyMap;
+import org.assertj.core.api.SoftAssertions;
+import org.jbehave.core.annotations.BeforeScenario;
+import org.jbehave.core.model.ExamplesTable;
+import org.jbehavesupport.core.TestContext;
+import org.jbehavesupport.core.expression.ExpressionEvaluatingParameter;
+import org.jbehavesupport.core.internal.parameterconverters.ExamplesEvaluationTableConverter;
+import org.jbehavesupport.core.internal.verification.ContainsVerifier;
+import org.jbehavesupport.core.internal.verification.NotContainsVerifier;
+import org.jbehavesupport.core.verification.Verifier;
+import org.jbehavesupport.core.verification.VerifierResolver;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.beans.factory.annotation.BeanFactoryAnnotationUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.VERIFIER;
+import static org.jbehavesupport.core.internal.ExamplesTableUtil.convertTable;
+import static org.springframework.util.Assert.isTrue;
+import static org.springframework.util.Assert.notNull;
+
+/**
+ * This class implements steps for testing ssh and provides customization option.
+ * This handler is registered as default. Register our own instance for customization.
+ * <p>
+ * Example:
+ * <pre>
+ * &#064;Configuration
+ * public class MyTestConfiguration {
+ *
+ *     &#064;Bean
+ *     public CustomSshHandler customSshHandler() {
+ *         return new CustomSshHandler();
+ *     }
+ *
+ * }
+ *
+ * public class CustomSshHandler extends SshHandler {
+ *
+ *     //project specific customization
+ *
+ * }
+ * </pre>
+ */
+
+@Slf4j
+@RequiredArgsConstructor
+public class SshHandler {
+    private final TestContext testContext;
+    private final  ConfigurableListableBeanFactory beanFactory;
+    private final  ExamplesEvaluationTableConverter tableConverter;
+    private final  VerifierResolver verifierResolver;
+    private final  ContainsVerifier containsVerifier;
+    private final  NotContainsVerifier notContainsVerifier;
```

Comment:
Decrease number of dependencies by pulling default verifiers into steps and present single entry point for `logContainsData` and `dataNotInLog`

Reply:
Done.

But I made dataNotInLog() method deprecated. As it is not needed in my point of view.

Reply:
If you agree i will remove it from documentation and tests.